src/main/java/com/mraof/minestuck/item/armor/IronLassArmorItem.java

```diff
@@ -0,0 +1,60 @@
+package com.mraof.minestuck.item.armor;
+
+import com.mraof.minestuck.util.MSParticleType;
+import com.mraof.minestuck.util.MSSoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ArmorItem;
+import net.minecraft.world.item.ArmorMaterial;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.Vec3;
+
+public class IronLassArmorItem extends ArmorItem
+{
+	public IronLassArmorItem(ArmorMaterial mat, ArmorItem.Type slot, Properties props)
+	{
+		super(mat, slot, props);
+	}
+	
+	@Override
+	public boolean canElytraFly(ItemStack stack, LivingEntity entity)
+	{
+		return true;
+	}
+	
+	@Override
+	public boolean elytraFlightTick(ItemStack stack, LivingEntity entity, int flightTicks)
+	{
+		if(entity instanceof Player player)
+		{
+			if(player.getItemBySlot(EquipmentSlot.FEET).getItem() instanceof IronLassArmorItem && player.isShiftKeyDown())
+			{
+				player.level().playSound(player, player.blockPosition(), MSSoundEvents.ITEM_JETPACK_FLIGHT.get(), SoundSource.PLAYERS, 1, player.getRandom().nextFloat()+0.35f);
+				player.level().addParticle(MSParticleType.EXHAUST.get(), player.getX(), player.getY(), player.getZ(), 0, 0, 0);
```

Comment:
what type of particles would you want?

Reply:
Same particle types as shoes seems fine, just something to indicate that its contributing

Reply:
that doesn't really make sense though, since the exhaust is generated from the feet doing the boosting, plus the feet can only boost during gliding anyways. not to mention that the current particles are already coming from the center of the player.

Reply:
I suppose the way I imagine it is that the chest piece is providing just enough thrust to allow for gliding, while the boots provide enough additional thrust to overcome gravity. That is why from my perspective it makes sense to always have just a little bit coming out during gliding regardless of whether the boots are in use or not. It wouldnt have to be every tick like the boots.

How were you imagining it?

Reply:
oh, you mean having the particles generate constantly! i thought you meant only during boosting for some reason. that makes a lot more sense actually, i can get this working pretty quick.